Lesson1/Selim_Osmanov/HT1_3(forward_list)/HT1_3(forward_list)/HT1_3main.cpp

```diff
@@ -0,0 +1,35 @@
+#include <iostream>
+#include <forward_list>
+
+#include "HT1_3head.hpp"
+
+template <typename T>
+void ShowElements ( std::forward_list <T> const & MyForwardList) {
+    std::cout << "My forward list is: [ ";
+    for(auto i : MyForwardList){
+        std::cout << i << " ";
+    }
+    std::cout << "]" << std::endl;
+}
+
+template <typename T>
+void ReverseElements (std::forward_list <T> & MyForwardList) {
+    std::forward_list <T> MyForwardList_copy = MyForwardList;
+    MyForwardList.clear();
+    for (auto element : MyForwardList_copy){
```

Comment:
element will be copied here.
plain auto in range for makes a copy.
you should use const reference here